src/manage/getAccessGrantAll.ts

```diff
@@ -50,7 +55,7 @@ async function getAccessGrantAll(
       requestor: string;
     }
   > = {},
-  options: AccessBaseOptions = {}
+  options: AccessBaseOptions & { includeExpiredGrants?: boolean } = {}
```

Comment:
```suggestion
  options: AccessBaseOptions & { includeExpired?: boolean } = {}
```

Maybe align on the name includeExpired?
We don't really need to say "Grant" given the context.